### PR TITLE
Add rhc-worker-bash in configs

### PR DESCRIPTION
**Pull request by r0x0d**
Patch to introduce the rhc-worker-bash in BaseOS/AppStream.

### CODE REVIEW

File: configs/sst_conversions.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    x86_64:
+    - rhc-worker-bash
+
+  # Add packages to the workload that don't exist (yet) in the repositories.
+  package_placeholders:
+    - srpm_name: rhc-worker-bash  # SRPM name (mandatory)
+      build_dependencies:   # build dependencies (RPM names) of this SRPM (optional)
+        - golang
+      limit_arches:         # can be left empty for all arches (optional)
+        - x86_64
```
Comment (jwboyer) [inline configs/sst_conversions.yaml:24]: Why is this x86_64 only?  The rhc package is offered on all architectures.

File: configs/sst_conversions.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+data:
+  name: RHC Worker Bash
+  description: Remote Host Configuration (rhc) worker for executing bash scripts on hosts managed by Red Hat Insights.
+  maintainer: sst_conversions
+
+  packages:
+    - rhc-worker-bash
+  
+  labels:     
+  - c9s # TODO(r0x0d): Verify if this is the correct label for RHEL 7
```
Comment (jwboyer) [inline configs/sst_conversions.yaml:12]: RHEL 7 doesn't have a CentOS Stream equivalent.  This is the correct label for anything going into CentOS Stream/RHEL 9.

Comment (bstinsonmhk) [inline configs/sst_conversions.yaml:12]: Is this only for EL7, or do you expect this to come in to EL8/EL9/EL10 too?

File: configs/sst_conversions.yaml (hunk 1 above)
Comment (r0x0d) [reply configs/sst_conversions.yaml:24]: Not exactly sure if the worker will have support for all other architectures. For now, I added x86_64 as a placeholder. Will confirm with other people to make sure what architectures we will be supporting.

File: configs/sst_conversions.yaml (hunk 2 above)
Comment (r0x0d) [reply configs/sst_conversions.yaml:12]: Only for EL7 for now.

Comment (r0x0d) [reply configs/sst_conversions.yaml:12]: Oh right, I just want to deploy this to EL7 for now, CentOS 7 will be needed as well, but not sure if we will need it there right away. @bocekm, would you know?

**Comment by bstinsonmhk** [comment]
This patch looks like an ok start whenever we're ready to ship this package in EL9, feel free to reopen this at that time.

**Comment by r0x0d** [comment]
> This patch looks like an ok start whenever we're ready to ship this package in EL9, feel free to reopen this at that time. 

This patch is not for EL9, it is for EL7. Should I create it in another repository or somewhere else? I get that the label was wrong too, but, could you point out to me the correct place to start out?

**Comment by bstinsonmhk** [comment]
There is nothing further that you need to do to work on that package in RHEL 7. This github repo only applies to EL9 and later.

File: configs/sst_conversions.yaml (hunk 2 above)
Comment (bocekm) [reply configs/sst_conversions.yaml:12]: We need the worker package in RHEL 7 only. We're working with the Software Production team to have the package copied over to a new unprotected CDN repository with every new version of the package (https://issues.redhat.com/browse/RHELDST-18108). We don't care if the package lands in CentOS Stream, Rocky, Alma, or other distros.

**Pull request closed without merging**